### ISSUE TITLE
Add config options to capture request headers

### ISSUE THREAD

**Issue by tommysitu** [SpectoLabs/hoverfly-java#118]
Hoverfly no longer capture request headers by default since the introduction of the v2 schema, see [Release v0.11.0] (https://github.com/SpectoLabs/hoverfly/releases/tag/v0.11.0)

I am proposing to introduce this config options:
```
      // Capture all headers
      configs().captureHeaders();

     // Capture a list of headers
     configs().captureHeaders("Authorization", "Origin");
```

This option will only take effect when Hoverfly is in capture mode.

**Comment by tommysitu** [comment]
Fixed in v0.6.2

**Comment by tommysitu** [comment]
Fixed in v0.6.2
